.changeset/hip-fireants-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql-toolbox": major
+---
+
+feat: Toolbox - major version bump to version 2.0.0
```

Comment:
Is this good enough? Do we not need to add context for why it's now 2.0.0?

Reply:
Nah it's not 😃 . I'll extend this.